pkg/arvo/app/spider.hoon

```diff
@@ -402,7 +490,24 @@
   =/  =tid  (yarn-to-tid yarn)
   =/  fail-cards  (thread-say-fail tid term tang)
   =^  cards  state  (thread-clean yarn)
-  [(weld fail-cards cards) state]
+  =^  http-cards  state  (thread-http-fail tid term tang)
+  [:(weld fail-cards cards http-cards) state]
+::
+++  thread-http-response
+  |=  [=tid =vase]
+  ^-  (quip card ^state)
+  =-  (fall - `state)
```

Comment:
How would we get an HTTP response from a nonexistent thread?  This seems like it should be an assertion, not a silent no-op.